pennylane/vqe/vqe.py

```diff
@@ -18,539 +18,9 @@
 # pylint: disable=too-many-arguments, too-few-public-methods
 from collections.abc import Sequence
 import warnings
-import itertools
-from copy import copy
 
 import pennylane as qml
 from pennylane import numpy as np
-from pennylane.operation import Observable, Tensor
```

Comment:
Maybe we could do a function called `Hamiltonian` here that just raises a deprecation warning and points users to the location in the ops folder?

Reply:
I added a class, using the example of `VQECost` which is just a dummy for `ExpvalCost`